### PR TITLE
Cr 1114 unrestricted agent

### PR DESCRIPTION
**Pull request by rpcatling**
# Motivation and Context
Following on from the **agentId** changes from string to integer in CC-API, changes are necessary here to fit.
See JIRA CR-1114.

This needs to be merged at the same time as the CCSvc PR  : https://github.com/ONSdigital/census-contact-centre-service/pull/99

### CODE REVIEW

File: src/test/resources/integrationtests/case/testCaseEndpoints.feature [hunk 1]
```diff
@@ -102,19 +102,17 @@ Feature: Test Contact centre Case Endpoints
     When I Refuse a case
     Then An error is thrown and no case is returned <httpError>
 
-    Examples: 
-      | caseId                                 | agentId  | httpError |
-      | "3305e937-6fb1-4ce1-9d4c-077f147789ab" | ""       | "400"     |
-      | "3305e937-6fb1-4ce1-9d4c-077f147789ab" | "ABC"    | "400"     |
```
Comment (PeterBochel) [inline src/test/resources/integrationtests/case/testCaseEndpoints.feature:108]: Isn't the agentId=ABC test is still valid?

File: src/test/resources/integrationtests/case/testCaseEndpoints.feature [hunk 2]
```diff
@@ -102,19 +102,17 @@ Feature: Test Contact centre Case Endpoints
...
-      | "3305e937-6fb1-4ce1-9d4c-077f147789ab" | "123456" | "400"     |
```
Comment (PeterBochel) [inline src/test/resources/integrationtests/case/testCaseEndpoints.feature:109]: Similar to previous comment, but shouldn't this now be hitting the endpoint with a number that is too big for an integer ?

File: src/test/resources/integrationtests/case/testCaseEndpoints.feature (hunk 1 above)
Comment (rpcatling) [reply src/test/resources/integrationtests/case/testCaseEndpoints.feature:108]: As we have discussed in our call with Phil, he doesn't see the cucumber test primary objective to construct malformed JSON (we can and do test some of this in the unit tests in CCSvc eg in CaseEndpointRefusalTest.java). 
So he is happy that these type of tests are removed from the cucumber.

File: src/test/resources/integrationtests/case/testCaseEndpoints.feature (hunk 2 above)
Comment (rpcatling) [reply src/test/resources/integrationtests/case/testCaseEndpoints.feature:109]: See comment above.

**Comment by rpcatling** [comment]
Hi Martin, yes, the removed types of tests are the domain on the unit testing. See the comment I added above in reply to Pete.

**Pull request merged**